### PR TITLE
Fix as many http:// images so plugin site doesn't have mixed content

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

**Comment by halkeye** [comment]
in addition:
https://github.com/jenkinsci/semantic-versioning-plugin/pull/37
https://github.com/jenkinsci/rhnpush-plugin/pull/8
https://github.com/jenkinsci/tekton-client-plugin/pull/237

File: gitlab-logo/README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 Display GitLab Repository Icon on dashboard
 
-[![](http://sebastian-badge.info/plugins/gitlab-logo.svg)](https://wiki.jenkins-ci.org/display/JENKINS/GitLab+Logo+Plugin)
-[![](https://jenkins.ci.cloudbees.com/buildStatus/icon?job=plugins/gitlab-logo-plugin)](https://jenkins.ci.cloudbees.com/job/plugins/job/gitlab-logo-plugin/)
+[![](https://img.shields.io/jenkins/plugin/v/gitlab-logo.svg)](https://plugins.jenkins.io/gitlab-logo)
+[![](https://jenkins.ci.jenkins.io/buildStatus/icon?job=plugins/gitlab-logo-plugin)](https://jenkins.ci.jenkins.io/job/plugins/job/gitlab-logo-plugin/)
```
Comment (MarkEWaite) [inline gitlab-logo/README.md]: Fix a broken URL
```suggestion
[![](https://ci.jenkins.io/buildStatus/icon?job=plugins/gitlab-logo-plugin)](https://ci.jenkins.io/job/plugins/job/gitlab-logo-plugin/)
```

File: scm-sync-configuration/README.md [hunk 2]
```diff
@@ -515,5 +515,5 @@ plugin](http://issues.jenkins-ci.org/browse/JENKINS-7762)
 # Greetings to our sponsors
 
 I'm a proud user of
-[![](http://www.jetbrains.com/idea/opensource/img/all/banners/idea120x30_white.gif)](http://www.jetbrains.com/idea/).  
+[![](docs/images/dea120x30_white.gif)](https://www.jetbrains.com/idea/).
```
Comment (MarkEWaite) [inline scm-sync-configuration/README.md]: Fix broken image
```suggestion
[![](docs/images/idea120x30_white.gif)](https://www.jetbrains.com/idea/).  
```

**Review by MarkEWaite — APPROVED** [review]
Two corrections that will fix broken items, but that are optional.  Approved even if those items are left broken.  Thanks for the improvement!

**Comment by halkeye** [comment]
Thanks so much Mark. It was a bunch of copy and pasting so I knew I'd miss something

I'll give it a few more hours then never

**Review by oleg-nenashev — COMMENTED** [review]
Thanks for the cleanup!

**Pull request merged**